lib/govuk_connect/cli.rb

```diff
@@ -113,6 +113,10 @@ def print_ssh_username_configuration_help
     info "Check this is correct, and if it isn't, set the `USER` environment variable to the correct username."
   end
 
+  def print_vpn_help_info
+    info "You may also wish to check that you are connected to the VPN, as attempting to SSH into an instance when NOT on the VPN can result in an operation timed out error"
```

Comment:
Nit ```suggestion
    info "Check that you are connected to the VPN, as not being on the VPN can result in an operation timed out error"
```

if you wanna make it more concise without losing meaning.